static-website-aws-python/__main__.py

```diff
@@ -0,0 +1,80 @@
+import pulumi
+import pulumi_aws as aws
+import pulumi_synced_folder as synced_folder
+
+# Import the program's configuration settings.
+config = pulumi.Config()
+path = config.get("path") or "./site"
+index_document = config.get("indexDocument") or "index.html"
+error_document = config.get("errorDocument") or "error.html"
+
+# Create an S3 bucket and configure it as a website.
+bucket = aws.s3.Bucket("bucket",
+    acl="public-read",
+    website=aws.s3.BucketWebsiteArgs(
+        index_document=index_document,
+        error_document=error_document,
+    ))
```

Comment:
Generally, PEP8 would have one of the following adjustments. It is *not* a blocker, and I'm not going to comment on every single one here because that's up to you:
All arguments at the same indent level, individual lines:
```suggestion
bucket = aws.s3.Bucket(
    "bucket",
    acl="public-read",
    website=aws.s3.BucketWebsiteArgs(
        index_document=index_document,
        error_document=error_document,
    ))
```
OR
All arguments aligned with the opening delimiter:
```suggestion
bucket = aws.s3.Bucket("bucket",
                       acl="public-read",
                       website=aws.s3.BucketWebsiteArgs(
                           index_document=index_document,
                           error_document=error_document,
                       ))
```

Reply:
Ok great, thanks. I think I'll go with your first suggestion, then, as that's also what I get out of the box when I run `black` against these files -- and I personally find that version a bit more readable as well. 😄

Reply:
"Readability counts." 😉